doc/index.rst

```diff
@@ -39,24 +39,28 @@ User Guide
 ==========
 
 The DSM CSC is a support CSC that is used in conjunction with the DSM UI in order
-to produce seeing measurements along the DSM probes line of sight. The functionality to produce these measurements are contained within the UI and are not described in this
-document. The UI is also responsible for the interaction with the DSM probe hardware. 
+to produce seeing measurements along the DSM probes line of sight.
+The functionality to produce these measurements are contained within the UI and are not described in this document.
+The UI is also responsible for the interaction with the DSM probe hardware. 
 The operational interaction of these systems and other details are captured in `SITCOMTN-001 <https://sitcomtn-001.lsst.io/>`_ technical note.
 
-There are two scripts that perform operations of the CSC. They are ``run_dsm.py`` and ``shutdown_dsm.py``. The parameters they take can be found by passing ``-h`` or
-``--help`` to the given script. The ``run_dsm.py`` script constructs the CSC and sends
-it to ``ENABLED`` state, allowing for the CSC to function straight away. It can be
-run in real mode or one of two simulation modes. The simulation modes will be shown
-in the next section. To run the CSC in real mode, do the following.
+There are two scripts that perform operations of the CSC.
+They are ``run_dsm.py`` and ``shutdown_dsm.py``. The parameters they take can be found by passing ``-h`` or ``--help`` to the given script.
+The ``run_dsm.py`` script constructs the CSC and using a flag, sends it to ``ENABLED`` state, allowing for the CSC to function straight away.
```

Comment:
I find this a bit confusing and even slightly misleading, since the user can specify other initials states. Please consider something along these lines:

The ``run_dsm.py`` script starts the CSC and optionally sends it to a specified state.
We typically want this CSC to be running straight away, in which case specify ``--state=enabled``.

Reply:
I like your way of putting it better, so that's what I used.